django_oemof_api/urls.py

```diff
@@ -17,8 +17,11 @@
 
 from django.contrib import admin
 from django.urls import path, include
+from .views import add_datapackage, delete_datapackage
 
 urlpatterns = [
+    path("admin/add_datapackage", add_datapackage, name="add_datapackage"),
```

Comment:
I though this would force the admin login, but it doesn't, do you have an idea how to enforce this without creating user database @henhuy ?

Reply:
I think you have to create a custom `AdminSite` in order to add a custom view...
See https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#adding-views-to-admin-sites
Never did this before.
Nevertheless, I think even for a admin user you will not get around migrating a user table...